### DISCUSSION TITLE
Filters with belongsTo

### DISCUSSION THREAD

**Discussion by edalzell** [statamic-rad-pack/runway#310] [Help]
We have a model the belongs to a country: 
<img width="611" alt="CleanShot 2023-08-03 at 15 20 47@2x" src="https://github.com/duncanmcclean/runway/assets/6069653/cdf0bc2d-6aab-478b-afbe-f78dbce2c6c0">

When we use the built in filter, we can only filter by country `id`.

I've made a custom filter that works but I see one of 2 feature requests:

1) if the field is a belongs to make it work magically, some how
2) allow us to disable the built in filters by field, so we don't have duplicates

My guess is that # 2 is much easier.

**Comment by duncanmcclean** [comment]
Let me know what you think of #313. It copies how filtering by fields on related entries works.
